dat/missions/flf/dvl/flf_dvl01.lua

```diff
@@ -0,0 +1,295 @@
+   You were too busy dodging incoming fire, rebalancing your shields, and generally trying to kill your attackers before they kill you to notice that Gregar, your passenger, has roused from his slumber. Clearly the noise and the rocking have jolted him awake. You snap at him not to distract you from this fight, but he desperately interrupts.
+   "These guys are my contacts, my friends! I was supposed to meet them here! Oh crap, this is not good. I didn't realize I'd be out this long! Look, I need to use your comm array right now. Trust me!"
+   Before you have a chance to ask him what he thinks he's doing, Gregar begins tuning your communications array, and soon finds the frequency he wants.
+   "FLF sentinel formation, this is Lt. Gregar, authorization code six-six-niner-four-seven-Gamma-Foxtrot! Cease fire, I repeat, cease fire!" He then turns to you. "Same to you. Stop shooting. This is a misunderstanding, they're not your enemies."]]))
+   tk.msg(_("Gregar puts an end to hostilities"), _([[You are skeptical at first, but a few seconds later it becomes apparent that the FLF fighters have indeed ceased firing. Then, there is an incoming comm from the lead ship.
+   "This is FLF sentinel Alpha. Lt. Gregar, authorization code verified. Why are you with that civilian? Where is your ship? And why didn't you contact us right away?"
+   "Apologies, Alpha. It's a long story. For now, let me just tell you that you can trust the pilot of this ship. This pilot kindly helped me out of a desperate situation, and without that help I probably would never have returned alive. Request you escort us to Sindbad."
+   "Copy that Lt. Gregar." Alpha then addresses you. "Please follow us. We will guide you to our base. Stay close. Sensor range is lousy in these parts, and if you get separated from us, we won't be able to find you again, and you won't be able to find us or our base."
+   With that, Alpha breaks the connection. It seems you have little choice but to do as he says if you ever want to take Gregar to his destination.]]))
+   faction.get("FLF"):setReputationGlobal( 5 ) -- Small buffer to ensure it doesn't go negative again right away.
+   misn.osdCreate(_("Deal with the FLF agent"), {
+      fmt.f( _("Take Gregar, the FLF agent, to the {sys} system and make contact with the FLF"), {sys=mem.destsys}),
+      _("Follow the FLF ships to their secret base. Do not lose them!"),
+      _("Alternatively, turn Gregar in to the nearest Dvaered base"),
+   })
+   misn.osdActive(2)
+   hook.timer(0.5, "inRange")
+end
+
+function inRange()
+   local mindist = 2000 -- definitely OOR.
+   for i, p in ipairs(fleetFLF) do
+      if p ~= nil and p:exists() then
+         mindist = math.min(mindist, vec2.dist(p:pos(), player.pos()))
+      end
+   end
+   if mindist < 500 then
+      hook.timer(2.0, "annai")
+      mem.OORT = hook.timer(10.0, "outOfRange")
+   else
+      hook.timer(0.5, "inRange")
+   end
```

Comment:
```suggestion
   local mindist = 2e3 -- definitely OOR.
   local ppp = player.pos()
   for i, p in ipairs(fleetFLF) do
      if p ~= nil and p:exists() then
         mindist = math.min(mindist, vec2.dist(p:pos(), ppp))
      end
   end
   if mindist < 500 then
      hook.timer(2, "annai")
      mem.OORT = hook.timer(10, "outOfRange")
   else
      hook.timer(0.5, "inRange")
   end
```

Minor formatting and optimization.